ast/misc.go

```diff
@@ -721,6 +723,43 @@ func (n *SetPwdStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+type ChangeStmt struct {
+	stmtNode
+
+	NodeType string
+	State    string
+	NodeID   string
+}
+
+// Restore implements Node interface.
+func (n *ChangeStmt) Restore(ctx *RestoreCtx) error {
+	ctx.WriteKeyWord("CHANGE")
+	ctx.WriteKeyWord(strings.ToUpper(n.NodeType))
+	ctx.WriteKeyWord("TO")
```

Comment:
```suggestion
	ctx.WriteKeyWord(" TO NODE_STATE ")
```

And delete the next line